gaspi-standard/tex/GASPI_Standard_Draft.tex

```diff
@@ -4420,9 +4433,9 @@ \subsubsection{\gaspifunction{write\_notify}}
 %Two new entries are inserted into the given queue.
 
 \begin{implementoradvice}
-The procedure is semantically equivalent to a call to \gaspifunction{write}
-and a subsequent call of \gaspifunction{notify}. However, it should be implemented
-more efficiently, if supported by the network infrastructure.
+The procedure is not semantically equivalent to a call to \gaspifunction{write}
+and a subsequent call of \gaspifunction{notify}. This call does not
+enforce an ordering relative to preceding write operations.

```

Comment:
Was the conclusion that there is not ordering wrt. _other_ write operations (rather than _preceding_)?